### ISSUE TITLE
Errors - Downloaded from Asset store getting Assets\LeanTween\Framework\LeanTweenExt.cs(31,166): error CS1503: Argument 3: cannot convert from 'System.Action' to 'Action'

### ISSUE THREAD

**Issue by DarkSeth** [dentedpixel/LeanTween#175]
HI. Looking for help to get this to work. Getting the following errors

Assets\LeanTween\Framework\LeanTweenExt.cs(31,166): error CS1503: Argument 3: cannot convert from 'System.Action' to 'Action'
Assets\LeanTween\Framework\LTDescrOptional.cs(62,21): error CS0037: Cannot convert null to 'Action' because it is a non-nullable value type
Assets\LeanTween\Framework\LTDescrOptional.cs(65,18): error CS0037: Cannot convert null to 'Action' because it is a non-nullable value type
Assets\LeanTween\Framework\LTSeq.cs(125,48): error CS1503: Argument 2: cannot convert from 'System.Action' to 'Action'

**Comment by basisbit** [comment]
The software developer of LeanTween stopped responding here at GitHub end of 2018

**Comment by DarkSeth** [comment]
Yeah. I see. BTW I get the error as soon as I install the new input system. Looks like I have to go DOtween route
